origin.spec

```diff
@@ -6,7 +6,7 @@
 %global gopath      %{_datadir}/gocode
 %global import_path github.com/openshift/origin
 
-%global golang_version 1.10
+%global golang_version 1.11
```

Comment:
Check with @sosiouxme on this change

Reply:
You probably will have to open a pr to ocp-build-data

Reply:
Will do.

Reply:
It's already there.

Reply:
apparently i forgot to comment here.
the spec file requesting 1.11 should suffice to get it.
image configs in ocp-build-data should already be updated for when this lands.